src/mocks/ScrollChainMockFinalize.sol

```diff
@@ -37,6 +33,19 @@ contract ScrollChainMockFinalize is ScrollChain {
         );
 
         // actions after verification
-        _afterFinalizeBatch(batchIndex, batchHash, totalL1MessagesPoppedOverall, postStateRoot, withdrawRoot);
+        _afterFinalizeBatch(batchIndex, batchHash, totalL1MessagesPoppedOverall, postStateRoot, withdrawRoot, true);
```

Comment:
Why do we use V1 queue and old public input format for enforced batches? The old prover cannot prove the new evm